koekalenteri-shared/model/Event.ts

```diff
@@ -0,0 +1,27 @@
+export type Event = {
+    id: string
+    "event-type": string
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/koekalenteri/koekalenteri/pullRequest?prid=7376473)
```suggestion
    'event-type': string
```